de_project/de_app/views.py

```diff
@@ -1,9 +1,15 @@
 from django.shortcuts import render
 
-# Create your views here.
+from .models import RecordHandler
 
 from django.http import HttpResponse
+from django.http import JsonResponse
 
 
 def index(request):
-    return HttpResponse("Hello, world!")
\ No newline at end of file
+    return HttpResponse("Hello, world!")
+
+def get_data_as_json_file(request, record_handler_name):
+    record_handler = RecordHandler.objects.get(name=record_handler_name)
+    json = record_handler.get_records_as_json()
+    return HttpResponse(json, content_type='application/json')
```

Comment:
As more and more records are added to individual record handlers, the response size of this method may reach a breaking point. I would recommend returning a paginated response rather than the entire contents of the record handler.

Reply:
Yes! We should definitely add a paginated option in another PR.